spec/services/stats/base_report_spec.rb

```diff
@@ -9,9 +9,8 @@ def report_type
   end
 
   describe BaseReport do
-
-    before(:all)  { create :report_type }
-    after(:all)   { ReportType.delete_all }
+    before(:all) { create_report_type(abbr: :r006) }
+    after(:all) { DbHousekeeping.clean(seed: true) }
```

Comment:
I decided to explicitly set the `seed` parameter as it is quite an important distinction (default true, can be set to false) and as a result was giving some false test outputs (for which tests had been adjusted!)